docs/pages/dsCompositeIn/ds-compositeIn_L3_TestProcedure.md

```diff
@@ -0,0 +1,392 @@
+          - "chmod +x /opt/HAL/dscompositeIn_L3/run.sh"
+          - cp -rf /usr/lib/libdshal.so /opt/HAL/dscompositeIn_L3/
+          - "ln -s /usr/lib/libds-hal.so /opt/HAL/dscompositeIn_L3/libdshal.so"
+        streams:
+        test1_VerifyConnect_Callback:
+          <<: *defaults
+          streams:
+        test2_VerifyStatus_Callback:
+          <<: *defaults
+          streams:
+        test3_ScaleAndVerify_Video:
+          <<: *defaults
+          streams:
+        test4_VerifySignal_Callback:
+          <<: *defaults
+          streams:
+```
+
+#### Test Suite Configuration
+
+Example Test Setup configuration File: `ut/host/tests/dsClasses/dscompositeIn_test_suite.yml`
+
+Update the execute command according to the device path where `HAL` binaries are copied.
+
+```yaml
+dscompositeIn:
+  description: "dscompositeIn Device Settings testing profile"
+  test:
+    execute: "/tmp/run.sh -p /tmp/Sink_compositeIn.yaml"
+    type: UT-C  # Cunit tests (UT-C)
+
+```
+
+## Run Test Cases
+
+Once the environment is set up, you can execute the test cases with the following command
+
+```bash
+python <TestCaseName.py> --config </PATH>/ut/host/tests/configs/example_rack_config.yml --deviceConfig </PATH>/ut/host/tests/configs/deviceConfig.yml
+```
+
+## Test Setup Connections
+
+To verify the compositeIn connect the external devices to `DUT`.
+For Example:
+
+- Connect external device with Composite Out to Composite In port.
+
+## Test Cases
+### dsCompositeIn_test1_VerifyConnect_Callback.py
+
+#### Platform Support - test01
+
+- Sink
+
+#### User Input Required - test01
+
+**Yes**: User interaction is necessary to connect/disconnect Composite Source device (This will be automated later).
+
+#### Acceptance Criteria - test01
+
+Connect the external device and check if dut recognize the event.
+Disconnect the external device and check if dut recognize the event.
+
+#### Expected Results - test01
+
+The test registers the event and check for  the event call back.
+
+Success Criteria
+
+- Device should recognize the connect/Disconnect event.
+
+#### Test Steps - test01
+
+- Initiate the Test:
+
+  - Select and execute the Python file: **`dsCompositeIn_test1_VerifyConnect_Callback.py`**
+  - The test will automatically download all required artifacts, copying them to the designated target directory before commencing execution.
+
+- Device Connect prompt:
+
+    The test will Request User to connect Source device to Composite In port:
+
+  - Question: "Connect the compositeIn source device to Port port_type and press Y: (Y/N)"
+  - Press **Y** if user connected the device and acknowledge the same. (this will mark the step as PASS).
+  - Press **N** if user could not connect the device. (this will mark the step as FAIL).
+
+- Device connect Confirmation:
+
+  - Test will check if the event has reached the device. 
+  - If event detected will mark the step as PASS
+  - If event not detected will mark the step as DAIL
```

Comment:
DAIL instead of FAIL. Typo. Please check through out the file

Reply:
done